gha/jira/basejiracommand.py

```diff
@@ -19,6 +19,12 @@
 DEFAULT_JIRA_SERVER = environ.get("JIRA_SERVER") or None
 DEFAULT_JIRA_TOKEN = environ.get("JIRA_TOKEN") or None
 QA_USER = environ.get("QA_USERNAME") or None
+SEARCH_QA_EXECUTE_JQL = """
+    project = "%(project)s"
+    AND type = "%(issuetype)s"
+    AND summary ~ "%(summary)s"
```

Comment:
Попробуй exact match здесь, если я правильно помню должно сработать такое:

```suggestion
    AND summary ~ "\"%(summary)s\""
```

Reply:
А это для чего?

Reply:
Без этого всё находилось правильно
![image](https://user-images.githubusercontent.com/32365962/209555302-6c48150e-9fd1-46a0-8d5e-91fcce661f30.png)

Reply:
Понял для чего... Но ни `=` ни твоё предложение не работаю. Ищу как должно быть. Сейчас просто проверю прямо в jql в жире

Reply:
Я указал - это exact match. Т.е. поиск по точному совпадению. Сейчас ты игнорируешь последнюю цифру тикета, т.е. "Проверка GBL-1" и "Проверка GBL-12" у тебя одно и то же.

Reply:
Попробуй добавить слэшей, заменить внешние двойные кавычки на одинарные (и тогда слэши не нужны).

Reply:
В общем всё работало правильно)
![image](https://user-images.githubusercontent.com/32365962/209555627-98515199-dd6e-4eba-8f0e-b85e5931ddd7.png)

Reply:
Это не тот случай.

Reply:
Поооонял. Да, с заменой кавычек сработало. Проверил подставив в jql запрос summary='Проверка задачи GBLPRG-73' - ничего не нашлось, а с правильным summary='Проверка задачи GBLPRG-737' нашёлся нужный тикет